tests/Component.Tests/project.json

```diff
@@ -9,6 +9,7 @@
 		"Components": "1.0.0-*",
 		"FluentAssertions": "4.0.0",
 		"Microsoft.Framework.DependencyInjection": "1.0.0-beta8",
+		"TestAttributes": "2.0.0-rc1",
 		"xunit": "2.1.0",
```

Comment:
No, we don't. That's half the point of this new system; dependency versions are not tracked.